Fixes at least devel/ccrtp's build:
  In file included from ccrtp/crypto/gcrypt/gcrypthmac.cpp:23:
  /usr/local/include/gcrypt.h:509: error: comma at end of enumerator list
  /usr/local/include/gcrypt.h:1346: error: comma at end of enumerator list

From: Werner Koch <wk@gnupg.org>
Date: Thu, 19 Mar 2015 09:43:55 +0000 (+0100)
Subject: Fix two pedantic warnings.
X-Git-Url: http://git.gnupg.org/cgi-bin/gitweb.cgi?p=libgcrypt.git;a=commitdiff_plain;h=f5832285b0e420d77be1b8da10a1e1d86583b414

Fix two pedantic warnings.

* src/gcrypt.h.in (gcry_mpi_flag, gcry_mac_algos): Remove trailing
comma.
--

Reported-by: Opal Raava <opalraava@hushmail.com>
Signed-off-by: Werner Koch <wk@gnupg.org>
---

--- src/gcrypt.h.in
+++ src/gcrypt.h.in
@@ -511,7 +511,7 @@ enum gcry_mpi_flag
     GCRYMPI_FLAG_USER1 = 0x0100,/* User flag 1.  */
     GCRYMPI_FLAG_USER2 = 0x0200,/* User flag 2.  */
     GCRYMPI_FLAG_USER3 = 0x0400,/* User flag 3.  */
-    GCRYMPI_FLAG_USER4 = 0x0800,/* User flag 4.  */
+    GCRYMPI_FLAG_USER4 = 0x0800 /* User flag 4.  */
   };
 
 
@@ -1372,7 +1372,7 @@ enum gcry_mac_algos
 /* Flags used with the open function.  */
 enum gcry_mac_flags
   {
-    GCRY_MAC_FLAG_SECURE = 1,  /* Allocate all buffers in "secure" memory.  */
+    GCRY_MAC_FLAG_SECURE = 1   /* Allocate all buffers in "secure" memory.  */
   };
 
 /* Create a MAC handle for algorithm ALGO.  FLAGS may be given as an bitwise OR
